doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1001,90 +911,61 @@ Induction
 
 .. seealso:: :tacn:`functional inversion`
 
-.. todo move the following label to just
-        before ":tacn:`discriminate` and :tacn:`injection`"
-
-.. _equality-inductive_types:
-
-Equality and inductive sets
----------------------------
-
-We describe in this section some special purpose tactics dealing with
-equality and inductive sets or types. These tactics use the
-equality :g:`eq:forall (A:Type), A->A->Prop`, simply written with the infix
-symbol :g:`=`.
-
 .. tacn:: decide equality
-   :name: decide equality
 
-   This tactic solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}`,
+   .. todo PR: why the curly braces below instead of parentheses?
+      shouldn't "+" be "/\"?
```

Comment:
No. `{ ... } + { ... }` is a notation for sum types defined in the standard library.

Reply:
A fairly big wart on the notation system is that it's very hard to find notation definitions.  CoqIDE should provide a way to do this.

In this case `Reserved Notation "{ A }  +  { B }" (at level 50, left associativity).` doesn't match the obvious regex (to me, after doing this a bunch) `Notation.*} \+ \{` because of the extra spaces.

Reply:
`Locate` can be used for this purpose. E.g., `Locate "{ _ } + { _ }".` gives `Notation "{ A } + { B }" := (sumbool A B) : type_scope (default interpretation)`. CoqIDE could indeed be improved to automatically do this `Locate` query when, e.g., hovering over a notation.